README.md

```diff
@@ -82,6 +82,20 @@ Ubuntu 18.04     | 5.26
 Zonemaster requieres Perl version 5.14.2 or higher. Zonemaster has been 
 tested with the default version of Perl in the OSs as listed in the table above.
 
+### Supported Client Browser versions
+
+The Zonemaster GUI is tested against these configurations and should work perfectly with close configuration.
+
+Operating System | Browser | Version
+---------------- | ------- | -------
+Ubuntu 18.04     | Firefox | 64+
+Ubuntu 18.04     | Chrome  | 64+
+Widnows          | Firefox | 64+
+Widnows          | Chrome  | 64+
```

Comment:
The versions should be the specific versions that we have tested (are going to test). In my Mac I have version 65 of Firefox and 73 of Chrome.